Infrastructure/PluginExecutor.cs

```diff
@@ -42,16 +48,26 @@ public void Run(string message, string user)
                     || CheckIfIlegalPluginPressed(message, out int pluginNumber, out msgForUser))
                 {
                     _messageSender.SendMessage(user, msgForUser);
-                    return;
+                    return "";
                 }
                 var extraData = ExtractExtraData(message);
                 var pluginType = ExtractPluginType(pluginNumber);
-
                 Execute(pluginType, extraData, user);
+                currentPluginId = _dal.LoadPluginData(user, SESSION_PLUGIN_ID);
+                var level =(currentPluginId!=null)? "\\" + currentPluginId:"";
```

Comment:
Instead of formatting the text here, we may return the current plugin id and the host (console\WhatsApp etc.) will decide how to format this

Reply:
ok